test/r2.txt

```diff
@@ -122,7 +122,7 @@ yum install /root/rpmbuild/RPMS/x86_64/cortx-prvsnr-2.0.0-0_git123456_el7.x86_64
 Step-4: Validate that `repos` states are available and visible for salt commands
 
 ```bash
-salt-call state.id_exists "configure pip" repos.pip_config
+salt-call '*' state.id_exists "configure pip" repos.pip_config
```

Comment:
`salt-call '*' ` won't work :)
Do update this to `salt '*'`

Reply:
Yes, I forgot to update the salt command. I fixed that.